hardware_interface/src/resource_manager.cpp

```diff
@@ -919,6 +912,38 @@ class ResourceStorage
     return hw_group_state_.at(group_name);
   }
 
+  /// Gets the logger for the resource storage
+  /**
+   * \return logger of the resource storage
+   */
+  rclcpp::Logger get_logger() const
+  {
+    if (logger_interface_)
+    {
+      return logger_interface_->get_logger().get_child("resource_manager");
+    }
+    else
+    {
+      return rclcpp::get_logger("resource_manager");
```

Comment:
I just repeat feedback I got in another PR, but shouldn't we use a static member for the logger instead of creating a logger at every call of get_logger()? Then we could pass the `logger` instead of the logger_interface and can create a persistent member variable in the actuator/sensor/system class like
`my_logger = logger.get_child("hardware_component.actor")` to append it to this one here?

Reply:
Yes! You are right! initially, I thought this needs to be done like this to have the logger services working. I made a small example this morning to test and it works this way as well. I've made the modifications in the code respectively.

Thank you!